interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -666,6 +695,62 @@ namespace cling {
   }
 
   Interpreter::CompilationResult
+  Interpreter::codeComplete(const std::string& line, size_t& cursor,
+                            std::vector<std::string>& completions) const {
+
+    const char * const argV = "cling";
+    std::string llvmdir = this->getCI()->getHeaderSearchOpts().ResourceDir;
+    std::string extra_part = "/lib/clang/3.9.0";
```

Comment:
The version number is not stable. You can see here http://clang.llvm.org/doxygen/CompilerInvocation_8cpp_source.html#l01329 how it's determined. Maybe just skip the trailing three subdirs? See `llvm::sys::Path`.